src/main/java/com/amazonaws/services/kinesis/clientlibrary/lib/worker/Worker.java

```diff
@@ -1495,7 +1497,7 @@ public Worker build() {
                     workerStateChangeListener,
                     shardSyncer,
                     leaderDecider,
-                    periodicShardSyncManager);
+                    null /* PeriodicShardSyncManager */);
```

Comment:
I think we dont need to provide an option to pass this in constructor. is there any use case?

Reply:
The use case would be for dependency injection for unit tests. Currently we don't have any unit tests where we're testing periodicShardSyncManager behavior (maybe we should?), but I figured we might want to have that option. I can remove it if you'd like.